gno.land/pkg/gnoweb/markdown/ext.go

```diff
@@ -0,0 +1,14 @@
+package markdown
+
+import (
+	"github.com/yuin/goldmark"
+)
+
+type gno struct{}
```

Comment:
this code is unclear; can you add a comment?

Reply:
Added a comment globally to the file